caddyhttp/httpserver/plugin.go

```diff
@@ -498,6 +498,7 @@ var directives = []string{
 	"grpc",      // github.com/pieterlouw/caddy-grpc
 	"gopkg",     // github.com/zikes/gopkg
 	"restic",    // github.com/restic/caddy
+	"nobots",    // github.com/Xumeiquer/nobots
```

Comment:
Are you sure the very end of the middleware chain is the proper place for your directive? Have you tried it in conjunction with `proxy` or `templates` or `browse`, for example?